docs/actions/execute-javascript.md

```diff
@@ -0,0 +1,188 @@
+---
+id: execute-javascript-server-side
+title: Execute JavaScript (Server-side)
+sidebar_label: Execute JavaScript (Server)
+---
+
+> Audience: [`Citizen Developer`](/docs/audience#software-developers)<br/>
```

Comment:
I would target this towards the Low-Code Engineer.

Reply:
I have changed the audience in this last commit. Also updated some broken links I hadn't seen before.